mne/io/curry/curry.py

```diff
@@ -0,0 +1,238 @@
+# -*- coding: UTF-8 -*-
+#
+# Authors: Dirk Gütlin <dirk.guetlin@stud.sbg.ac.at>
+#
+#
+# License: BSD (3-clause)
+
+import os
+import re
+import numpy as np
+import mne
+
+from ..base import BaseRaw, _check_update_montage
+from ..meas_info import create_info
+from ..utils import _read_segments_file, warn, _find_channels, _create_chs
+
+
+def _read_curry_events(fname_base, curry_vers):
+    """
+    Read curry files and convert the data for mne.
+    Inspired by Matt Pontifex' EEGlab loadcurry() extension.
+
+    :param full_fname:
+    :return:
+    """
+
+    #####################################
+
+    EVENT_FILE_EXTENSION = {7: ['.cef', '.ceo'], 8: ['.cdt.cef', '.cdt.ceo']}
+
+    event_file = None
+    for ext in EVENT_FILE_EXTENSION[curry_vers]:
+        if os.path.isfile(fname_base + ext):
+            event_file = fname_base + ext
+
+    if event_file is not None:
+        curry_events = []
```

Comment:
I don't understand the logic of this block. If those tokens are always there, the pseudocode of this block should be something like this:

```py
with open(event_file) as fid:
   _ = [line = next(fid)  while not line == 'NUMBER_LIST START']
   descriptions = [next(fid) while not line == 'NUMBER_LIST END']
   _ = [line = next(fid)  while not line == 'NUMBER_LIST START_LIST']
   data = [next(fid) while not line == 'NUMBER_LIST END_LIST']

return _parse_events(data, descriptions)
```

Reply:
Plus I've a question: whats the difference between `NUMBER_LIST` and `REMARK_LIST`?

Reply:
@DiGyt is this addressed?

Reply:
Sorry for having to wait. I wanted to make sure first, that the data is all read correcty.
Concerning this function, i changed the entire thing to an independent `read_events_curry` public function , that reads similar to `mne.read_events`